iamproperty/iamproperty.github.io
# Basic tables

## Summary of Changes:
1. Introduce the start of the tables element
2. Document the basic table

## Rationale
Start with the basic table format then expand later with more complicated demo's

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/tables

## Ticket(s)
DS-44

## Checklist

## Notes
This is the first pull request of a series of them which will expand the functionality of tables.
